### PR TITLE
make enabled collectors configurable (based on code from node_exporter)

### PR DESCRIPTION
**Pull request by martinlindhe**
Ping @carlpett

### CODE REVIEW

**Comment by martinlindhe** [comment]
This also includes other minor changes to make the code better match the node_exporter command

File: collector/os.go [hunk 1]
```diff
@@ -25,105 +29,90 @@ type OSCollector struct {
...
 
 // NewOSCollector ...
-func NewOSCollector() *OSCollector {
+func NewOSCollector() (Collector, error) {
 
 	return &OSCollector{
 
 		PagingMaxBytes: prometheus.NewDesc(
-			prometheus.BuildFQName(wmiNamespace, "os", "paging_max_bytes"),
+			prometheus.BuildFQName(Namespace, "os", "paging_max_bytes"),
```
Comment (carlpett) [inline collector/os.go]: `const subsystem = "os"`

**Comment by carlpett** [comment]
Except the small nitpick, looks really good!

**Comment by martinlindhe** [comment]
Thanks, I constified the "os" string. Landing it after build passed

**Pull request merged**